app/scripts/directives/util.js

```diff
@@ -178,4 +178,24 @@ angular.module('openshiftConsole')
         }
       });
     };
+  })
+  .directive('persistTabState', function($routeParams, $location) {
```

Comment:
Probably worth a comment saying `reloadOnSearch` should be false for the route to use this directive.